lib/checks.sh

```diff
@@ -448,13 +448,27 @@ function check_firewall() {
         _check_service_is_not_running 'Network' 'iptables'
     fi
 }
+function check_firewall_ports(){
+	local localip=$(ip -4 route get 8.8.8.8 | awk {'print $7'} | tr -d '\n')
+	local serviceports=( 88 389 636 3268 3269 )
+	local portstatus
 
+	for port in ${serviceports[@]}; do
+		portstatus=$(bash -c 'exec 3<> /dev/tcp/'${localip}'/'${port}';echo $?' 2>/dev/null)
```

Comment:
@alanjumpi This check would check if the firewall ports are open on the local host. The test should be against the AD/ldap server that someone is trying to do a direct-to-ad integration test with.

These tests would always fail, unless you are running it on a server with these ports open: 88 389 636 3268 3269.